common-app-lib/src/cmr/common_app/services/search.clj

```diff
@@ -55,17 +77,63 @@
   (fn [context query results]
     [(:concept-type query) (qm/base-result-format query)]))
 
-(defn find-concepts
-  "Executes a search for concepts using the given query."
-  [context concept-type query]
-  (validate-query context query)
+(defn- add-scroll-results-to-cache
+  "Adds the given search results (truncated to ony the :hits, :timed-out, and :scroll-id keys) 
```

Comment:
Is this going to work for any response format?

Reply:
Also nitpick, typo, **ony** instead of **only**

Reply:
Typo fixed. Discussing whether or not to eliminate support for ISO since the files are so large.